cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlIntegrationTestUtils.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Comment:
Nit: As a convention, CC repo uses the year the file has been created in copyright: `2017` -> `2021`.